docs/tutorials/openfoam/spack-openfoam.yaml

```diff
@@ -124,17 +124,17 @@ deployment_groups:
         destination: setup_openfoam.sh
         content: |
           #!/bin/bash
-          source /apps/spack/share/spack/setup-env.sh
+          source /opt/apps/spack/share/spack/setup-env.sh
           spack env activate openfoam
-          chmod -R a+rwX /apps/spack/var/spack/environments/openfoam
+          chmod -R a+rwX /opt/apps/spack/var/spack/environments/openfoam
```

Comment:
I believe this chmod line is no longer needed.

Reply:
I am ok if this happens in a separate PR if you want to keep this change focused.

Reply:
Updated.